parser/auto.go

```diff
@@ -67,6 +67,13 @@ func init() {
 		Help:     "Parse Juniper telemetry in the form of protocol buffers. Typicially combined with the UDP receiver.",
 		AutoMake: true,
 	})
+	Auto.Add(skogul.Module{
+		Name:     "protobuf_usp",
+		Aliases:  []string{"usp"},
+		Alloc:    func() interface{} { return &P{} },
+		Help:     "Parse Usp telemetry in the form of protocol buffers. Typicially combined with the UDP receiver.",
```

Comment:
So far we haven't got a single use case where it is combined with UDP though. Also: It should probably mention how it's an USP message contained by an USP record, since I for one have no idea whether this is the norm, or just what our hardware happen to do.